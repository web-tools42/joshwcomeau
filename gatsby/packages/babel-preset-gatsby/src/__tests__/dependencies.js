const preset = require(`../dependencies`)

expect.addSnapshotSerializer(require(`../utils/path-serializer`))

describe(`dependencies`, () => {
  it(`should specify proper presets and plugins`, () => {
    expect(preset()).toMatchSnapshot()
  })
})
