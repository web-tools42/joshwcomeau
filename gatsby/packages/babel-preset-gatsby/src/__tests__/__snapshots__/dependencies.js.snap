// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`dependencies should specify proper presets and plugins 1`] = `
Object {
  "plugins": Array [
    Array [
      "<PROJECT_ROOT>/node_modules/@babel/plugin-transform-runtime/lib/index.js",
      Object {
        "absoluteRuntime": "<PROJECT_ROOT>/node_modules/@babel/runtime",
        "corejs": false,
        "helpers": true,
        "regenerator": true,
        "useESModules": true,
      },
    ],
    "<PROJECT_ROOT>/node_modules/@babel/plugin-syntax-dynamic-import/lib/index.js",
  ],
  "presets": Array [
    Array [
      "<PROJECT_ROOT>/node_modules/@babel/preset-env/lib/index.js",
      Object {
        "corejs": 2,
        "exclude": Array [
          "transform-typeof-symbol",
        ],
        "modules": false,
        "useBuiltIns": "usage",
      },
    ],
  ],
  "sourceType": "unambiguous",
}
`;
